src/main/native/cpp/jni/NetworkTablesJNI.cpp

```diff
@@ -99,6 +109,14 @@ JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *reserved) {
   classNotFoundEx = JException(env, "java/lang/ClassNotFoundException");
   if (!classNotFoundEx) return JNI_ERR;
 
+  // Register natives using inner classes (works around JDK-8145897)
+  JClass cls(env, "edu/wpi/first/wpilibj/networktables/NetworkTablesJNI");
```

Comment:
This class being cleaned up when OnLoad exits is going to be ok, right? We don't need to hold onto it?